src/actions/profile-view.js

```diff
@@ -480,6 +481,31 @@ export function selectActiveTabTrack(
   };
 }
 
+/**
+ * An action for selecting the main track of active tab view directly.
+ * This is handy for some cases where we need to select the main track
+ * without actually clicking on it (e.g. closing event of resources panel).
+ */
+export function selectActiveTabMainTrack(): ThunkAction<void> {
+  return (dispatch, getState) => {
+    const globalTracks = getActiveTabGlobalTracks(getState());
```

Comment:
This all could be computed in `toggleResourcesPanel`, without a separate action.